src/config.rs

```diff
@@ -250,10 +250,68 @@ pub(crate) struct MentionsEntryConfig {
 
 #[derive(PartialEq, Eq, Debug, serde::Deserialize)]
 #[serde(rename_all = "kebab-case")]
-#[serde(deny_unknown_fields)]
 pub(crate) struct RelabelConfig {
     #[serde(default)]
     pub(crate) allow_unauthenticated: Vec<String>,
+    // alias identifier -> labels
+    #[serde(flatten)]
+    pub(crate) configs: Option<HashMap<String, RelabelRuleConfig>>,
```

Comment:
Let's maybe call it `aliases`, since that's what we are calling it everywhere else.
```suggestion
    pub(crate) aliases: HashMap<String, RelabelRuleConfig>,
```